packages/zkdb/src/sdk/authentication/authentication.ts

```diff
@@ -1,24 +1,24 @@
-import { AppContainer } from 'src/container.js';
+import { IApiClient } from '@zkdb/api';
 import storage from '../../storage/storage.js';
 import { SignedData } from '../../types/signing.js';
 import { Signer } from '../signer/interface/signer.js';
 import { ZKDatabaseUser } from '../types/zkdatabase-user.js';
 
 export class Authenticator {
```

Comment:
Instead of
```ts
private getSigner(){
    return this.signer;
}```

You can write:
```ts
private get signer(){
    return this.#signer;
}
```